server/main.go

```diff
@@ -147,21 +147,28 @@ func run() {
 		MinVersion:   tls.VersionTLS12,
 	})
 
+	grpcLogDeciderFunc := func(methodFullName string, err error) bool {
+		if err == nil && methodFullName == "/grpc.health.v1.Health/Check" {
```

Comment:
unsure if the leading slash is required:
```
"grpc.service":"grpc.health.v1.Health"
```

Reply:
Tried in the mini-lab, it's required.

Reply:
Here's one of their tests for this function: https://github.com/grpc-ecosystem/go-grpc-middleware/blob/be4c235f9d1c9c4efad8dd2bad3cce712eabdeb0/logging/logrus/server_interceptors_test.go#L249